arbitrum-docs/run-arbitrum-node/data-availability-committees/03-deploy-mirror-das.md

```diff
@@ -163,9 +167,8 @@ daserver
     --data-availability.s3-storage.secret-key "<YOUR SECRET KEY>"
     --data-availability.s3-storage.object-prefix "<YOUR OBJECT KEY PREFIX>/"
     --data-availability.s3-storage.discard-after-timeout false
-    --data-availability.local-db-storage.enable
-    --data-availability.local-db-storage.data-dir /home/user/data/badgerdb
-    --data-availability.local-db-storage.discard-after-timeout false
+    --data-availability.local-file-storage.enable
+    --data-availability.local-file-storage.data-dir /home/user/data/das-data
```

Comment:
The full set of options is now 
```
      --data-availability.local-file-storage.data-dir string                                       local data directory
      --data-availability.local-file-storage.enable                                                enable storage/retrieval of sequencer batch data from a directory of files, one per batch
      --data-availability.local-file-storage.enable-expiry                                         enable expiry of batches
      --data-availability.local-file-storage.max-retention duration                                sto
re requests with expiry times farther in the future than max-retention will be rejected (default 504h0m0s)
```

Reply:
Added them to the list of "Local files" parameters in https://arbitrum-docs-git-update-das-page-offchain-labs.vercel.app/run-arbitrum-node/data-availability-committees/deploy-das#step-2-deploy-the-das
(click the "Local files" button to see them)